tests/projects/schema.py

```diff
@@ -296,6 +313,16 @@ class ProjectConnection(ListConnectionWithTotalCount[ProjectType]):
     """Project connection documentation."""
 
 
+ProjectFeedItem = Annotated[
+    Union[IssueType, MilestoneType], strawberry.union("ProjectFeedItem")
+]
+
+
+@strawberry.type
+class ProjectFeedConnection(relay.Connection[ProjectFeedItem]):
+    pass
```

Comment:
Is this a left over from a test attempt? =P

This is not being used, so it should be removed